### ISSUE TITLE
pandoc path : unable to spawn process

### ISSUE THREAD

**Issue by electron0zero** [atom-community/markdown-preview-plus#204]
while using pandoc in windows(10 pro) when we set in MPP settings (as mentioned in install page [https://github.com/Galadirith/markdown-preview-plus/blob/master/docs/installation.md](url) ) atom throws a error saying unable to spawn process.

steps to reproduce 
1. run atom as admin.
2. install MPP
3. install pandoc
4. setup pandoc as mentioned in Docs.
5. open a markdown file ( .md ) 
6. Toggle preview 
:bomb: error thrown

Fix. 1
1. go to system environment variables (not local user environment  variables ) and add a path to pandoc executable folder.
2. remove path to pandoc from MPP settings page.
:smile: :smile_cat: it's working 

Fix. 2
skip step 4 while installing (don't setup path for pandoc in MPP)
:smile: it's working 

we can update setup docs [https://github.com/Galadirith/markdown-preview-plus/blob/master/docs/installation.md](url) to include this instructions for windows systems

**Comment by electron0zero** [comment]
closing this issue due to **no response for a long time**

**Comment by lierdakil** [comment]
Terribly sorry about that. We try to maintain MPP to the best of our
ability, but life tends to get in the way. Is this still relevant? If so, I
could update the docs. Even better if you could make a PR.

>

**Comment by electron0zero** [comment]
@lierdakil it's ok, you don't have to be sorry about it.

It was not a big Issue and it had a simple workaround so I forgot about it.
I reported this because I wanted everyone to know workaround of this issue.

**why close it now?**
I was looking on my issues tab and saw this was Idle for a long so I closed this.

**Is this issue still there?**
Can not say, I moved to VS Code from atom.
I will try to reproduce and report back asap
